frontend/src/app/bounty/create/page.tsx

```diff
@@ -57,7 +54,17 @@ const CreateBountyPage: FC = () => {
     const [imgLink, setImgLink] = useState("");
     const [filename, setFilename] = useState<string | undefined>();
 
+    const [minDeadline, setMinDeadline] = useState<Date>(new Date());
```

Comment:
I would avoid relying on `new Date()` to fetch the current time according to the user's browser.

```suggestion
    const [minDeadline, setMinDeadline] = useState<Date>();
```

Reply:
Good catch! Thanks!